internal/commands/login/inputs.go

```diff
@@ -66,19 +71,23 @@ func (i *inputs) Resolve(profile *user.Profile, ui terminal.UI) error {
 func (i *inputs) resolveCloudCredentials(u user.Credentials) []*survey.Question {
```

Comment:
note: only prompting for `CloudCredentials` (not `LocalCredentials`) when browser is launched. 
This is because we want the prompts to act as a guide for the user after they create api keys through the browser.